src/event/event.rs

```diff
@@ -173,6 +173,28 @@ impl Event {
         sys::event::is_aio(&self.inner)
     }
 
+    /// Returns true if the event contains mach port readiness.
+    ///
+    /// # Notes
+    ///
+    /// Method is available on all platforms, but not all platforms support mach ports.
+    ///
+    /// The table below shows what flags are checked on what OS.
+    ///
+    /// | [OS selector] | Flag(s) checked |
+    /// |---------------|-----------------|
+    /// | [epoll]       | *Not supported* |
+    /// | [kqueue]<sup>1</sup> | `EVFILT_MACHPORT` |
+    ///
+    /// 1: Only supported on MacOS.
```

Comment:
nit: I believe RustDoc supports Markdown footnotes, so you can actually write:
```suggestion
    /// | [kqueue][^1] | `EVFILT_MACHPORT` |
    ///
    /// [^1]: Only supported on MacOS.
```